Cesium3DTiles/src/BingMapsRasterOverlay.cpp

```diff
@@ -235,7 +242,7 @@ namespace {
  * {
  *   "imageryProviders": [
  *     {
- *       "attribution": "� 2021 Microsoft Corporation",
+ *       "attribution": "� 2021 Microsoft Corporation",
```

Comment:
Was this change intentional?

Reply:
I changed it back to the copyright sign but it still marks the line as changed even though I can't tell what the difference is. I even tried copy-pasting the old version back in, but git doesn't detect that anything's changed. Do you have any idea what might be going on?